src/commands/graph/graph-edit.js

```diff
@@ -0,0 +1,89 @@
+const process = require('process')
+
+const gitRepoInfo = require('git-repo-info')
+
+const {
+  createCLISession,
+  createPersistedQuery,
+  ensureAppForSite,
+  generateSessionName,
+  loadCLISession,
+  updateCLISessionMetadata,
+} = require('../../lib/one-graph/client')
+const {
+  defaultExampleOperationsDoc,
+  getNetligraphConfig,
+  readGraphQLOperationsSourceFile,
+} = require('../../lib/one-graph/netlify-graph')
+const { NETLIFYDEVERR, chalk } = require('../../utils')
+const { openBrowser } = require('../../utils/open-browser')
+
+const graphEdit = async (options, command) => {
+  const { api, site, state } = command.netlify
+  const siteId = site.id
+
+  if (!site.id) {
+    console.error(
```

Comment:
please use the `error` function from the `utils` folder that ends the command as well. (no need for process.exit)